### PR TITLE
Remove not supported statement as discussed internally

### PR DESCRIPTION
**Pull request by bdeitte**

### CODE REVIEW

**Review by ianpittwood — CHANGES_REQUESTED** [review]
We'll want to extend these changes to the READMEs located in the product directories too as they have the same "not formally supported" snippet in there. We may also want to get #461 merged prior to removing the warnings? There's a small change I know I need to make with the Justfiles that Karl requested, but it should otherwise be ready for review.

**Comment by bdeitte** [comment]
@ianpittwood Yes I forgot this was in the other READMEs, good call and doing that now.  And I will wait for the #461 merge

File: README.md [hunk 1]
```diff
@@ -4,17 +4,17 @@ Docker images for RStudio Professional Products
 
 **IMPORTANT:** There are a few things you need to know before using these images:
 
-1. These images are provided as a convenience to RStudio customers and are not formally supported by RStudio. If you
-   have questions about these images, you can ask them in the issues in the repository or to your support
-   representative, who will route them appropriately.
+1. This image may introduce **BREAKING** changes; as such we recommend:
+   - Avoid using the `latest` or `{operating-system}` tags to avoid unexpected version changes, and
+   - Always read through the NEWS.md file in the appropriate sub-folder to understand the changes before updating.
```
Comment (bdeitte) [inline README.md:9]: I removed the "not supported" statement but also added the BREAKING statement, felt appropriate to have this here as well while changing to mention sub-folders.

File: connect/README.md [hunk 2]
```diff
@@ -27,12 +27,9 @@ https://www.rstudio.com/products/connect/.
...
 # Notice for support
 
-1. This image may introduce **BREAKING** changes, as such we recommend:
+1. This image may introduce **BREAKING** changes; as such we recommend:
    - Avoid using the `latest` or `{operating-system}` tags to avoid unexpected version upgrades, and
-   - Always read through the [NEWS](./NEWS.md) to understand these changes before updating.
-1. These images are provided as a convenience to RStudio customers and are not yet formally supported by RStudio. If you
-   have questions about these images, you can ask them in the issues in the repository or to your support
-   representative, who will route them appropriately.
+   - Always read through the [NEWS](./NEWS.md) to understand the changes before updating.
```
Comment (bdeitte) [inline connect/README.md:32]: Removed "not supported" in all places, and some nit-picky grammar updates.

File: workbench-for-microsoft-azure-ml/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # RStudio Workbench for Microsoft Azure ML
 
-This image is built specifically for use with Azure ML.
+This image is built specifically for use with Azure ML. Please contact Posit support in order to use.
```
Comment (bdeitte) [inline workbench-for-microsoft-azure-ml/README.md:3]: As I looked at the READMEs, realizing we should add this given a separate license and some more info is needed to use, don't want anyone thinking they can just directly use this on Azure ML without help.

**Review by ianpittwood — APPROVED** [review]
LGTM! I just finished the updates to #461 as well so that's ready for reviews.

File: README.md (hunk 1 above)
Comment (colearendt) [reply README.md:9]: This reminds me that we should delete that tag for sure, since it is the default.

Comment (bdeitte) [reply README.md:9]: Oh didn't know that, will update!

**Pull request merged**